grails-app/services/org/pih/warehouse/requisition/RequisitionService.groovy

```diff
@@ -210,7 +210,29 @@ class RequisitionService {
      * @return
      */
     def getRequisitions(Requisition requisition, Map params) {
-        return getRequisitions(requisition, params, [], [])
+        List<Location> origins = [];
+        List<Location> destinations = [];
+
+        if (params.origin.getClass().isArray()) {
```

Comment:
Move it to the controller as:
```
def origins = params.origin ? Location.findAllByIdInList(params.list("origin")) : []
def destinations = params.destination ? Location.findAllByIdInList(params.list("destination")) : []
```
And then call the `getRequisitions` instead of `getAllRequisitionTemplates`:
`def requisitions = requisitionService.getRequisitions(requisition, params, origins, destinations)`